backend/testfiles/execution/db.tests

```diff
@@ -487,6 +487,17 @@ module PartialEvaluation =
   (friends (fun p -> let x = { y = { z = { a = [1;2;3;4;5] }}}
                      (x.y.z.a |> List.length_v0) < p.height)) = [" Chandler "; "GrumpyCat"; "Rachel"; "Ross" ]
 
+  // tuple destructuring outside query
```

Comment:
I think we can skip the test outside the query.

Reply:
done